src/backend/api/reviews.js

```diff
@@ -0,0 +1,86 @@
+const express = require("express");
+const reviewsRouter = express.Router();
+const dbcon = require("../database");
+
+// Returns all reviews.
+reviewsRouter.get("/", async (request, response) => {
+  let reviews = undefined;
+  try {
+    reviews = await dbcon("Review");
+  } catch (error) {
+    return response.status(500).send(error);
+  }
+  if (reviews.length < 1) {
+    return res.status(204).send();
```

Comment:
If your intention is to send an empty object/array here, `send([ ] / { })` makes more sense.